MVCapplication/src/main/java/com/softserve/if072/mvcapp/dto/UserRegistrationForm.java

```diff
@@ -0,0 +1,59 @@
+package com.softserve.if072.mvcapp.dto;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+public class UserRegistrationForm {
```

Comment:
Maybe for maintainability better to have only one class for login and registration